### PR TITLE
Remove incorrect `override` keywords

### PR DESCRIPTION
**Pull request by 333fred**
Fixes https://github.com/dotnet/docs/issues/26115. @AlekseyTs for review.

### CODE REVIEW

**Comment by AlekseyTs** [comment]
> ### Overrides in interfaces

Probably all references to "overrides" should be adjusted, not just in code examples.

---
Refers to: proposals/csharp-8.0/default-interface-methods.md:101 in a3cb067. [](commit_id = a3cb067b31a0fecc794a8a0a328b9c6ff537fae1, deletion_comment = False)

File: proposals/csharp-8.0/default-interface-methods.md [hunk 1]
```diff
@@ -109,11 +109,11 @@ interface IA
 }
 interface IB : IA
 {
-    override void IA.M() { WriteLine("IB.M"); } // explicitly named
+    void IA.M() { WriteLine("IB.M"); } // explicitly named
 }
 interface IC : IA
 {
-    override void M() { WriteLine("IC.M"); } // implicitly named
+    void M() { WriteLine("IC.M"); } // implicitly named
```
Comment (AlekseyTs) [inline proposals/csharp-8.0/default-interface-methods.md:116]: >void M() { WriteLine("IC.M"); } // implicitly named [](http://example.com/codeflow?start=4&length=51)

This declares a new method in IC that has no relation to IA.M and doesn't implement it. I think the original example meant to express syntax for implicit implementation. There is no syntax that can be used for an implicit implementation in a derived interface. It looks like we can simply remove the IC interface from this example.

**Comment by AlekseyTs** [comment]
> ### Overrides in interfaces

Interface methods cannot be overridden, they can only be implemented.

---
In reply to: [1068534052](https://github.com/dotnet/csharplang/pull/5828#issuecomment-1068534052) [](http://example.com/codeflow?ancestors=1068534052)

---
Refers to: proposals/csharp-8.0/default-interface-methods.md:101 in a3cb067. [](commit_id = a3cb067b31a0fecc794a8a0a328b9c6ff537fae1, deletion_comment = False)

**Comment by AlekseyTs** [comment]
Done with review pass (commit 1)

**Comment by 333fred** [comment]
@AlekseyTs please take another look.

File: proposals/csharp-8.0/default-interface-methods.md [hunk 2]
```diff
@@ -109,23 +109,23 @@ interface IA
...
 
-Public `virtual` function members in an interface may be overridden in a derived interface explicitly (by qualifying the name in the override declaration with the interface type that originally declared the method, and omitting an access modifier).
+Public `virtual` function members in an interface may be implemented in a derived interface explicitly (by qualifying the name in the override declaration with the interface type that originally declared the method, and omitting an access modifier).
 
-`virtual` function members in an interface may only be overridden explicitly (not implicitly) in derived interfaces, and members that are not `public` may only be implemented in a class or struct explicitly (not implicitly). In either case, the overridden or implemented member must be *accessible* where it is overridden.
+`virtual` function members in an interface may only be implemented explicitly (not implicitly) in derived interfaces, and members that are not `public` may only be implemented in a class or struct explicitly (not implicitly). In either case, the overridden or implemented member must be *accessible* where it is overridden.
```
Comment (AlekseyTs) [inline proposals/csharp-8.0/default-interface-methods.md:124]: It looks like there is still a reference to "overridden" term in this paragraph.

File: proposals/csharp-8.0/default-interface-methods.md [hunk 3]
```diff
@@ -109,23 +109,23 @@ interface IA
...
-    override void M() { WriteLine("IC.M"); } // implicitly named
+    void M() { WriteLine("IB.M"); } // Creates a new M, unrelated to `IA.M`. Warning
 }
 ```
 
-Override declarations in interfaces may not be declared `sealed`.
+Explicit implementations in interfaces may not be declared `sealed`.
 
-Public `virtual` function members in an interface may be overridden in a derived interface explicitly (by qualifying the name in the override declaration with the interface type that originally declared the method, and omitting an access modifier).
+Public `virtual` function members in an interface may be implemented in a derived interface explicitly (by qualifying the name in the override declaration with the interface type that originally declared the method, and omitting an access modifier).
```
Comment (AlekseyTs) [inline proposals/csharp-8.0/default-interface-methods.md:122]: >override declaration [](http://example.com/codeflow?start=134&length=20)

Consider rephrasing as well

File: proposals/csharp-8.0/default-interface-methods.md [hunk 4]
```diff
@@ -189,9 +189,9 @@ abstract class E : IA, IB, IC // ok
 }
 ```
 
-> ***Open issue***: should we support explicit interface abstract overrides in classes?
+> ***Open issue***: should we support explicit interface abstract implementations in classes?
 
-In addition, it is an error if in a class declaration the most specific override of some interface method is an abstract override that was declared in an interface. This is an existing rule restated using the new terminology.
+In addition, it is an error if in a class declaration the most specific override of some interface method is an abstract implementation that was declared in an interface. This is an existing rule restated using the new terminology.
```
Comment (AlekseyTs) [inline proposals/csharp-8.0/default-interface-methods.md:194]: >the most specific override [](http://example.com/codeflow?start=54&length=26)

Consider rephrasing

**Review by AlekseyTs — APPROVED** [review]
LGTM (commit 2)

**Comment by 333fred** [comment]
@AlekseyTs could you take another look?

**Comment by Chrobob-777** [comment]
can-ram-all-devices.pilots.jets

**Review by Chrobob-777 — COMMENTED** [review]
can_view/all_devices.pilots.jets[System](system.pilots.jets)

**Pull request merged**